src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,99 @@
+const expensesServices = require('../services/expenses.services');
+const usersServices = require('../services/users.services');
+
+const { asyncHandler } = require('../utils/asyncHandler.js');
+const { isExpenseValid } = require('../utils/expenses.js');
+const { getErrorWithStatus } = require('../utils/getError.js');
+
+const getExpenses = async (req, res) => {
+  const { categories, userId, from, to } = req.query;
+
+  let categoriesArray = categories;
+
+  if (categories) {
+    categoriesArray = [categories];
```

Comment:
The current implementation of `categoriesArray` assumes that `categories` is a single value. If `categories` can be a comma-separated list or an array, this logic should be adjusted to handle multiple categories correctly.